OverlayPlugin.Core/PluginMain.cs

```diff
@@ -247,29 +247,37 @@ public void InitPlugin(TabPage pluginScreenSpace, Label pluginStatusText)
                             // ** Init phase 2
                             this.label.Text = "Init Phase 2: Integrations";
 
-                            // Initialize the parser in the second phase since it needs the FFXIV plugin.
-                            // If OverlayPlugin is placed above the FFXIV plugin, it won't be available in the first
-                            // phase but it'll be loaded by the time we enter the second phase.
-                            _container.Register(new FFXIVRepository(_container));
-                            _container.Register(new NetworkParser(_container));
-                            _container.Register(new TriggIntegration(_container));
-                            _container.Register(new FFXIVCustomLogLines(_container));
-                            _container.Register(new MemoryProcessors.AtkStage.FFXIVClientStructs.Data(_container));
-
-                            // Register FFXIV memory reading subcomponents.
-                            // Must be done before loading addons.
-                            _container.Register(new FFXIVMemory(_container));
-
-                            // These are registered to be lazy-loaded. Use interface to force TinyIoC to use singleton pattern.
-                            _container.Register<ICombatantMemory, CombatantMemoryManager>();
-                            _container.Register<ITargetMemory, TargetMemoryManager>();
-                            _container.Register<IAggroMemory, AggroMemoryManager>();
-                            _container.Register<IEnmityMemory, EnmityMemoryManager>();
-                            _container.Register<IEnmityHudMemory, EnmityHudMemoryManager>();
-                            _container.Register<IInCombatMemory, InCombatMemoryManager>();
-                            _container.Register<IAtkStageMemory, AtkStageMemoryManager>();
-
-                            _container.Register(new OverlayPluginLogLines(_container));
+                            // Wrap FFXIV plugin related initialization in try/catch to allow OP to work when FFXIV plugin isn't present
+                            try
+                            {
+                                // Initialize the parser in the second phase since it needs the FFXIV plugin.
+                                // If OverlayPlugin is placed above the FFXIV plugin, it won't be available in the first
+                                // phase but it'll be loaded by the time we enter the second phase.
+                                _container.Register(new FFXIVRepository(_container));
```

Comment:
Is this something we could early out in here once instead, rather than individually treating these pieces as optional?

Reply:
This is functionally an early out via exception, because the second register here (for `NetworkParser`) throws an exception.

Ultimately I'd like to completely split it out so that the components that rely on FFXIV_ACT_Plugin sit in their own event source and can be initialized separately, but there's some intertwined logic in MiniParseEventSource's PartyChange logic which makes that difficult.

I guess a better question would be "should this PR be canned in favor of a much larger PR that does significant refactoring to separate the FFXIV-specific OP code out from the doesn't-require-FFXIV code?"

Reply:
Yeah, fair.  This seems fine to land now and can be replaced by a larger PR later.

I think my concern is that I don't understand how to prevent this behavior from reverting (in any automated test sense).  Having a larger PR to separate things makes it more likely that this won't happen again, because there will be more of a separation of concerns.